apps/app/src/components/profile/my-feeds/hooks/useFilteredFeeds.ts

```
@@ -0,0 +1,31 @@
+) {
+  return feeds.filter((feed) => {
+    const matchesSearch =
+      !searchTerm ||
+      feed.name.toLowerCase().includes(searchTerm.toLowerCase()) ||
+      (feed.description &&
+        feed.description.toLowerCase().includes(searchTerm.toLowerCase()));
+
+    // Check if feed setup is complete based on config
+    const isComplete = !!(
+      feed.config &&
+      feed.config.name &&
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)